python/lang/security/audit/sqli/asyncpg-sqli.py

```diff
@@ -64,6 +64,14 @@ def bad10(conn: asyncpg.Connection):
         # ruleid: asyncpg-sqli
         cur = await conn.cursor(sql_query)
 
+def bad11(conn: Connection):
+    import common
+    sql_query = common.bad_query_1.format(user_input)
+    # ruleid: asyncpg-sqli
+    cur = conn.fetch(sql_query)
+    # ruleid: asyncpg-sqli
+    cur = conn.fetch(common.bad_query_1.format(user_input))
```

Comment:
This behavior requires interfile support which is a Semgrep Pro feature. Rules in this registry should be compatible with OSS Semgrep which does not support interfile analysis.

Reply:
Roger - thank you!

The problematic `cur = conn.fetch(sql_query)` code on Line 71 is NOT detected by Semgrep Pro. I think we might need another (Pro perhaps?) pattern to flag it.

Thanks for the help.

Reply:
I have removed the code block which was generating a false negative.

The pattern added in this PR detects the corresponding bad code block just fine.

Thanks for the review help.

Reply:
Thanks for the flag on the Pro miss - will send this to the team